apps/cnquery/cmd/providers.go

```diff
@@ -51,9 +52,17 @@ var listProvidersCmd = &cobra.Command{
 var installProviderCmd = &cobra.Command{
 	Use:    "install <NAME>",
```

Comment:
I think we should use the formal [provider-name]@[version-number]. This has the advantage that we do not need to validate if --version and --file or --url are conflicting

Reply:
makes sense, i changed the code